resolwe/flow/utils/test.py

```diff
@@ -224,6 +224,8 @@ def tearDown(self):
                 shutil.rmtree(data_dir, ignore_errors=True)
                 shutil.rmtree(export_dir, ignore_errors=True)
 
+        Process.objects.all().delete()
```

Comment:
Look like that manual delete works, but Django follows a bit different procedure in `tearDown` which crashes because of this anomaly.